app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -68,7 +68,7 @@ def save_inventory_multi(association, hashes, deletes, find_key, child_keys = []
   def save_inventory_single(type, parent, hash, child_keys = [], extra_keys = [], disconnect = false)
     child = parent.send(type)
     if hash.blank?
-      disconnect ? child.disconnect_inv : child.try(:destroy)
+      disconnect ? child.try(:disconnect_inv) : child.try(:destroy)
```

Comment:
Is this a fix for what @zeari is handing in: #8329 ? (`undefined method 'disconnect_inv' for nil:NilClass`).
This could be a valid alternative.